code/style.css

```diff
@@ -1,4 +1,213 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  margin: 0;
+  background: white
 }
+
+
+header {
+  display: block;
+  height: 200px;
+  background-color: #ffffff;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}                                 
+
+
+.logo-img {
+  width: 250px;
+  margin: 0 0 0 2%;                    
+}
+
+
+nav {
+  align-self: flex-end;                    
+  margin: 14px;
+}
+
+
+.topnav {
+  background-color: #ffff;
+  overflow: hidden;
+  font-family: 'Bebas Neue', cursive;
+  
+}
+
+
+.topnav a {
+  float: left;
+  color: #c0c0c0;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 25px;
+}
+
+
+.topnav a:hover {
+  background-color: #c59000;
+  color: #393e46;
+}  
+
+
+article {
+  background: #FFF8F0; 
+  padding: 20px;
+}
+
+
+.big-news {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+
+.big-news-img {
+  width: 100%;
+  margin: 10px;
+  border-radius: 43px;
+  background: #e0e0e0;
+  box-shadow:  20px 20px 31px #9b9b9b,
+               -20px -20px 31px #ffffff;
+}
+
+
+h1 {   
+  color: #A68D65;
+  margin: 10px;
+  font-family: 'Bebas Neue', cursive;
+  font-size: 88px;
+}
+
+
+.small-news {
+  background-color: white;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  object-fit: none;              
+  align-items: center;
+}
+
+
+.news-box {
+  display: block;
+  margin: 20px 20px 10px 20px;
+  width: 550px;
+  border-radius: 43px;
+  background: #e0e0e0;
+  box-shadow:  20px 20px 31px #9b9b9b,
+             -20px -20px 31px #ffffff;
+}
+
+
+h3 { /*Text to small news-box*/
+  font-family: 'Bebas Neue', cursive;
+  margin: 10px 0 30px 0; 
+}
+
+
+.small-news a {
+  float: left;
+  color: #A68D65;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: unset;
+  font-size: 20px;
+}
+
+
+.small-news a:hover {
+color: black;
+}
+
+
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  background-color: #A68D65; 
+  height: 160px;
+  padding: 20px;
+}
+
+
+h4 { /*text for footer*/
+  font-family: 'Bebas Neue', cursive;
+  color: #FFF8F0;
+}
+
+
+@media (min-width: 668px) {
+  header { 
+    justify-content: space-around;
+    flex-direction: column;
+ }      
+} 
+  
+  
+@media (min-width: 668px){
+  nav {
+    align-self: center; 
+ }
+}  
+
+
+@media (min-width: 668px) {
+h1 {
+ font-size: 110px; 
+ margin: 0 0 50px 0;
+ }
+}
+
+
+@media (min-width: 668px) {
+.small-news {
+ flex-flow: wrap; 
+ }
+}
+
+
+@media (min-width: 668px) and (max-width: 1023px) {
+.news-box {
+ width: 300px; 
+ }
+}
+
+
+@media (min-width: 1024px) {
+.news-box {
+  width: 350px;
```

Comment:
I think this is "tossed around" and may have caused you some headache?  If you changes the width property for your .nexs-box (row 95) to 350px and in the media query for bigger screens, set the width to 550px and then it works perfectly!